.probo.yaml

```diff
@@ -69,7 +69,7 @@ steps:
       - chown www-data:www-data -R /var/www/html/sites/default/files
```

Comment:
Do you want to remove that `zaporylie/composer-drupal-optimizations` from line 18 in this file as well?